fstat/static/js/snackbar.js

```diff
@@ -0,0 +1,17 @@
+$("body").prepend('<div id="snackbar"></div>')
```

Comment:
This change is separate from this I believe. You should probably remove to avoid merge conflicts.